src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationViewItem.cs

```diff
@@ -879,11 +879,15 @@ internal void PropagateDepthToChildren(int depth)
 			}
 		}
 
-		internal void OnExpandCollapseChevronTapped(object sender, TappedRoutedEventArgs args)
-		{
-			IsExpanded = !IsExpanded;
-			args.Handled = true;
-		}
+		// UNO - OnExpandCollapseChevronTapped is not necessary because NavigationViewMenuItem explicitly
```

Comment:
That is likely masking another issue, because this is a ported code.

I'm okay with it as a workaround as long as it doesn't break anything, but we will need to find a better fix in future.